<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>53</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Загрузка</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>12</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="value">
    <number>-1</number>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="format">
    <string>%v/%m</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
